patches/minecraft/net/minecraft/core/RegistrySetBuilder.java.patch

```diff
@@ -1,5 +1,20 @@
 --- a/net/minecraft/core/RegistrySetBuilder.java
 +++ b/net/minecraft/core/RegistrySetBuilder.java
+@@ -28,6 +_,14 @@
+       };
+    }
+ 
++   static <T> HolderLookup.RegistryLookup<T> wrapRegistryLookup(final HolderLookup.RegistryLookup<T> lookup) {
++      return new HolderLookup.RegistryLookup.Delegate<>() {
++         @Override protected RegistryLookup<T> m_254893_() { return lookup; }
++         @Override public Stream<HolderSet.Named<T>> m_214063_() { return Stream.empty(); }
++         @Override public Optional<HolderSet.Named<T>> m_254901_(TagKey<T> key) { return Optional.of(HolderSet.m_255229_(lookup, key)); }
++      };
++   }
++
```

Comment:
Is there any need for this to be a patch instead of in a Forge class like `DataPackRegistriesHooks`?

Reply:
There is not, I just added it under the vanilla method. I can move it though.